Platform Documentation.md

```diff
@@ -339,6 +339,8 @@ $ cctrlapp APP_NAME/DEP_NAME deploy
 To deploy a specific version, append your version control systems identifier (full commit-SHA1 for Git or an integer for Bazaar).
 If not specified, the version to be deployed defaults to the latest image available (the one built during the last successful push).
 
+Running [workers](#Workers) have to be redeployed separately by initiating a [manual restart](#restarting-workers).
```

Comment:
I think it would make more sense to move this after the next paragraph. The next paragraph explains how the web deploy works and how the switchover between old and new version is done without downtime. The note about workers having to be restarted manually to get the new version makes still sense after that.